modules/microshift-storage-classes.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-classes_{context}"]
+== Storage classes
+
+Storage classes provide the workload layer interface for selecting a device class. The following storage class parameters are supported in {product-title}:
+
+* The `csi.storage.k8s.io/fstype` parameter selects the filesystem types. Both `xfs` or `ext4` filesystem types are supported.
+* The `topolvm.io/device-class` parameter is the name of the device class. If a device class is not provided, the default device class is assumed.
+
+Multiple storage classes can refer to the same device class. You can provide varying sets of parameters for the same backing device class, such as `xfs` and `ext4` variants.
+
+.{product-title} defines the following as the default storage class:
+
+[source, yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true" <1>
+  name: topolvm-provisioner
+parameters:
+  "csi.storage.k8s.io/fstype": "xfs" <2>
+provisioner: topolvm.io <3>
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer <4>
+allowVolumeExpansion: <5>
+----
+[.small]
+<1> An example of {product-title}'s default storage class. If a PVC does not specify a storage class, this class is assumed. There can only be one default storage class in a cluster. Having no value assigned to this annotation is also supported.
```

Comment:
I'd reword to avoid "Microshift's". I think you could probably just say "An example of the default storage class."